.github/workflows/build_images.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build OCI Images
+
+on:
+  push:
+    branches:
+      - master
+      - gh_workflow
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the ci image
+        run: docker build . --file Containerfile --target ci-base --tag pulp/pulp-ci:latest
+      - name: Build the single-container image
+        run: docker build . --file Containerfile --tag pulp/pulp-fedora31:latest
+      - name: Greet on gh_workflow
```

Comment:
```suggestion
      - name: Docker login
        env:
          DOCKER_BOT_PASSWORD: ${{ secrets.DOCKER_BOT_PASSWORD }}
          DOCKER_BOT_USERNAME: ${{ secrets.DOCKER_BOT_USERNAME }}
        run: echo "$DOCKER_BOT_PASSWORD" | docker login -u "$DOCKER_BOT_USERNAME" --password-stdin docker.io
      - name: Greet on gh_workflow
```

Reply:
I added DOCKER_BOT_USERNAME and DOCKER_BOT_PASSWORD as secrets in GHA for this repo.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

Reply:
Are those secrets limited to being available on master push actions?